src/goto-programs/remove_const_function_pointers.cpp

```diff
@@ -12,15 +12,16 @@ Author: Thomas Kiley, thomas.kiley@diffblue.com
 #include "remove_const_function_pointers.h"
 
 #include <util/arith_tools.h>
+#include <util/format_expr.h>
 #include <util/simplify_expr.h>
 #include <util/std_expr.h>
 #include <util/symbol_table.h>
 
 #include "goto_functions.h"
 
 #define LOG(message, irep) \
```

Comment:
If this is changed to multiple statements it should be in a block, otherwise it'll break in strange ways when doing something like `if(something) LOG(something_else, e)`.

Reply:
Done, thanks for picking this up!